package.json

```diff
@@ -14,7 +14,6 @@
     "@babel/preset-typescript": "^7.1.0",
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.1.0",
-    "@emotion/babel-plugin": "^11.9.2",
```

Comment:
shouldn't the yarn lock file be updated?

Reply:
The babel plugin is also a transitive dependency, so the lockfile isn't updated in this case